frontend/src/components/Gallery/_Gallery.scss

```diff
@@ -1,76 +1,85 @@
 .search {
-    &_gallery {
-        margin-top: 20px;
-        display: grid;
-        grid-template-columns: repeat(2, 1fr);
-        grid-gap: 1em;
-        
-        &--image {
-            width: 100%;
-            height: 100%;
-            max-width: 100%;
-            min-height: 100%;
-            object-fit: cover;
-        }
+  &_gallery {
+    margin-top: 20px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 1em;
 
-        &--no-image {
-            margin-top: 50px;
-            display: flex;
-            justify-content: center;
-        }
+    &--image {
+      width: 100%;
+      height: 100%;
+      max-width: 100%;
+      min-height: 100%;
+      object-fit: cover;
+    }
 
-        &--no-image-text {
-            color: $white;
-            font-size: 30px;
-        }
+    &--no-image {
+      margin-top: 50px;
+      display: flex;
+      justify-content: center;
     }
-    
-    &_gallery > div {
-        overflow: hidden;
-        position: relative;
-        height: 42vw;
-        border-radius: 8px;
-    }    
+
+    &--no-image-text {
+      color: $white;
+      font-size: 30px;
+    }
+  }
+
+  &_gallery > div {
+    overflow: hidden;
+    position: relative;
+    height: 42vw;
+    border-radius: 8px;
+  }
+}
+.expand-button {
+  display: none;
+  background-color: $darkBlue;
```

Comment:
I know maybe I am asking a lot :) but when I checked it on the environment, seems like `$pink` is more visible for the user. `$darkBlue` is not visible in some dark images. Could you change this too?

Reply:
```suggestion
  background-color: $pink;
```